example_school_library_decorator/book.rb

```diff
@@ -0,0 +1,9 @@
+class Book
+  attr_accessor :title, :author, :rentals
```

Comment:
- [ ] Please use `attr_reader` for `:rentals`  variable since it belongs to the `Rental` class and can not be written directly to in the Book Class.